schemas/create-ami-set-request.yml

```diff
@@ -0,0 +1,39 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      Create AMI Set Request
+description: |
+  Create a new AMI Set
+type:                       object
+properties:
+  amis:
+    type: array
+    description: |
+      AMIs in this set
+    items:
+      type: object
+      properties:
+        region:
+          type: string
+          description: |
+            The Amazon AWS region of the AMI set
+        hvm:
+          type: string
+          description: |
+            The AMI that uses HVM virtualization type
+        pv:
+          type: string
+          description: |
+            The AMI that uses PV virtualization type
+      additionalProperties: false
+      required:
+        - region
+        - hvm
+        - pv
+  lastModified:
```

Comment:
Sorry that I didn't see this before, but we also should not have last modified as an allowed field in the request.  we will generate this value inside the api handler, so forcing a value here that we won't end up using isn't ideal.